src/components/Table/Table.css

```diff
@@ -3,15 +3,14 @@
 .container {
   min-width: 100%;
   clear: both;
-  @media only screen and (max-width: 960px) {
-    margin: 0 -25px;
+  @media only screen and (max-width: 700px) {
```

Comment:
We should use this for everything - https://github.com/odota/ui/blob/master/src/components/App/App.css#L20 like here, not only for table